spec/unit/routing_spec.rb

```diff
@@ -15,7 +15,7 @@
   let(:namespaces) { ActiveAdmin.application.namespaces }
 
   it "should only have the namespaces necessary for route testing" do
-    expect(namespaces.names).to eq [:admin]
+    expect(namespaces.names).to eq [[:admin]]
```

Comment:
If Namespace::Store#names now returns a set of arrays instead of symbols I suppose it should be deprecated and replaced by Namespace::Store#name_paths